src/auto_scheduler/search_policy/utils.cc

```diff
@@ -414,19 +414,52 @@ void PruneInvalidState(const SearchTask& task, Array<State>* states) {
   }
 }
 
+/********** SplitFactorizationMemo **********/
+
+void SplitFactorizationMemo::ReadWriteLock::GetRead() {
```

Comment:
Better to comment the read condition (no one is writing).

Reply:
Updated the doc string as well as added the comments.